lib/client.ts

```diff
@@ -165,6 +165,20 @@ export default class Client {
       .then(res => res.richmenus);
   }
 
+  public setDefaultRichMenu(richMenuId: string): Promise<any> {
```

Comment:
Return type may be `Promise<{}>` or `Promise<void>`. Please choose what you prefer.